http-verbs-common/src/main/scala/uk/gov/hmrc/http/HttpPatch.scala

```diff
@@ -16,7 +16,7 @@
 
 package uk.gov.hmrc.http
 
-import play.api.libs.json.{Json, Writes}
+import _root_.play.api.libs.json.{Json, Writes}
```

Comment:
If we add package `uk.gov.hmrc.http.play` (to correct the current `uk.gov.hmrc.play.http`), it will also require clients who do `import uk.gov.hmrc.http._` to use `_root_` for play stuff.

Reply:
We've moved to `client2`